src/equations/linearized_euler_2d.jl

```diff
@@ -143,6 +161,187 @@ end
     end
 end
 
+@inline function max_abs_speed_naive(u_ll, u_rr, normal_direction::AbstractVector,
+                                     equations::LinearizedEulerEquations2D)
+    @unpack v_mean_global, c_mean_global = equations
+    v_mean_normal = normal_direction[1] * v_mean_global[1] +
+                    normal_direction[2] * v_mean_global[2]
+    return abs(v_mean_normal) + c_mean_global * norm(normal_direction)
+end
+
+@inline function flux_upwind(u_ll, u_rr, orientation::Integer,
```

Comment:
Can you please add a docstring with a scientific reference (see other examples, e.g., for 2D Euler) (also to the other flux).

Reply:
Done in [f1f4ae4](https://github.com/trixi-framework/Trixi.jl/pull/1557/commits/f1f4ae41c3e5b7c0ce3171262586b7b7c10be45a). Had some unrelated issues while trying to create the docs locally so we have to see if it renders as intended.